include/atomics/receiverCadmium.hpp

```diff
@@ -7,7 +7,6 @@
 #ifndef BOOST_SIMULATION_PDEVS_RECEIVER_HPP

```

Comment:
The following rule for file names should be followed: Use all lowercase to avoid operating system conflict.
so receiverCadmium.hpp can be changed to receiver_cadmium.hpp.
The same file name change should be done for other files with capital letters.
The files that include these files should also be modified.

Reply:
thanks for the catch. I have renamed all the files now. Please review latest code